build/helpers/ifEqual.js

```diff
@@ -0,0 +1,10 @@
+/* https://github.com/zeke/handlebars-helper-equal/blob/master/index.js */
+module.exports = function(lvalue, rvalue, options) {
+  if (arguments.length < 3)
```

Comment:
Leaving it as-is, yes. I followed the example from http://fbrctr.github.io/advanced/helpers.html which shows helper functions being defined in this manner.

BUT, there's nothing stopping us from taking a different approach where multiple helpers are defined in a single module that exports an object, which can then be passed along as:

```js
assemble({
  helpers: require('./build/helpers')
});
```

OR we could use `requireDir()` and keep the helpers in single files, which is cleaner IMO.